src/main/java/spoon/reflect/visitor/chain/CtQueryImpl.java

```diff
@@ -80,9 +80,7 @@ public CtQueryImpl addInput(Object... input) {
 			this.inputs = new ArrayList<>();
 		}
 		if (input != null) {
-			for (Object in : input) {
-				this.inputs.add(in);
-			}
+			Collections.addAll(this.inputs, input);
```

Comment:
Why using the static call instead of using `this.inputs.addAll(input)`?

Reply:
https://m.youtube.com/watch?v=15X0qFtBqiQ

Reply:
And I think that static call shows what is added to what in a clearer way.

Reply:
I agree with using `addAll` instead of the loop: my question was only about why not using the invocation from the instance directly.
IMHO it's more readable than calling `Collections.addAll` with two arguments.

Reply:
I tried to refactor from **Collections.addAll()** to **this.input.addAll(input)**, but second variant cannot be compiled because of argument type which is: addInput(Object... input)

**Collections.addAll()** is:
```addAll(Collection<? super T> c, T... elements)```

**this.input.addAll()** is:
```addAll(Collection<? extends E> c)```

The exact mismatch in **this.input.addAll(input)** is:
```
addAll in List (java.util.Collection<?>) cannot be applied to (java.lang.Object[])
```

In other words: **Object... input** is an array, so it can be used only in Collection<? super T> c, **T... elements**)

So the only way in this context is static method Collections.addAll().

Reply:
OK for this one then. But for the other ones you could have use the form instance.addAll() no?

Reply:
I used explicit and simple way in all cases: x.addAll(Arrays.asList(y)) :-)
No need for Collections.addAll()
I am ready to merge.